internal/server/server.go

```diff
@@ -88,7 +88,7 @@ func createSOMASServer(
 				shared.Speaker:   0,
 			},
 			SpeakerID:          shared.Team1,
-			JudgeID:            shared.Team2,
+			JudgeID:            shared.Team4,
```

Comment:
Nooo... Make sure you don't change this.

Reply:
@Spruudis We should probably request to have this changed to some random assignment. It's a pretty big advantage to start with a role.

Reply:
can't we only revert it when creating PR to the main branch? it's just a waste of time and effort (I know it's a small one) to revert this, create local PR and then change it back again for testing. 
if you have a very strong opinion about it, then I'll do it 😉

Reply:
I do kinda have a strong opinion about it :P
Firstly this is only used when testing the Judge, so there is no use in having it on the client. Secondly, if we start checking in code which we shouldn't touch we might forget and either assume an outcome that is not the case or forget to fix it later

Reply:
ok, I'll fix it